package.json

```diff
@@ -32,6 +32,7 @@
     "unit": "bash lighthouse-core/scripts/run-mocha.sh --default",
     "closure": "cd lighthouse-core && node closure/closure-type-checking.js",
     "devtools": "bash lighthouse-core/scripts/roll-to-devtools.sh",
+    "compdevtools": "bash lighthouse-core/scripts/compile-against-devtools.sh",
```

Comment:
sure sure. 
i found and installed `https://github.com/mklabs/yarn-completions` which does completions for our scripts on `yarn run` so i'm happy now. :)